### DISCUSSION TITLE
[Feature Request] Multiple TX, Multiple RX solution

### DISCUSSION THREAD

**Discussion by bennett-dahl** [ExpressLRS/ExpressLRS#1409] [Ideas]
Not sure this is the right way to Open this kind of request, but it seems commonplace in much of the community. Feel free to direct me elsewhere if need be.

I found a similar question asked here:[https://www.reddit.com/r/ExpressLRS/comments/sexkth/controlling_multiple_quads_with_2_tx/hunluxl/?context=3](url)

the long and short is basically having multiple radios, and multiple quads, and wanting to be able to connect whichever to each SIMULTANEOUSLY, easily for the purpose of flying with friends. The idea was to use model match to differentiate, 

I did the testing and verified what u/bingo-fuel speculated. Using model match doesn't work this way.

I'm not sure the ins and outs of how the system works, but is there a way to change the binding phrase without re-flashing? I feel like a simple (from a user standpoint) solution would be to be able to read and change the binding phrase of a connected rx and the tx, together or independently, through the Lua. As a failsafe if you mix things up, or cant remember the binding phrase, you could do a "traditional bind" to get them talking, then using the LUA change both of them.

This also seems like a decent solution if you get to a race day, and two people happen to have the same binding phrase. Instead of needing to reflash both the tx and rx, you could update them "on the spot" to some extent. I'd also just love to be able to grab whatever radio, and whatever quad and/or multiple and go fly, solo, or with friends.

**Comment by piwi3910** [comment]
don't use a binding phrase

**Comment by schugabe** [comment]
@rockclimber399 I moved the issue to the ideas discussions area.

The problem of "two people with the same binding phrase" is unlikely since it has no default value and as long as the string is somewhat complex it's very unlikely that this happens. 

Does traditional binding help with your usecase? You could flash both tx modules with two different bind phrases and the rx without bind phrase. Than way you can re-bind when needed
